src/s1reader/s1_burst_slc.py

```diff
@@ -360,6 +389,68 @@ class represents a polynomial function of range
 
         return az_carrier_poly
 
+    def total_doppler(self, xstep=500, ystep=50):
+        """
+        Compute total Doppler which is the sum of two components: 
+        (1) the geometrical Doppler induced by the relative movement 
+        of the sensor and target
+        (2) the TOPS specicifc Doppler caused by the electric steering 
+        of the beam along the azimuth direction resulting in Doppler varying
+        with azimuth time. 
+        Parameters
+        ----------
+        xstep: int
+            Spacing along x direction [pixels]
+        ystep: int
+            Spacing along y direction [pixels]
+
+        Returns
```

Comment:
Modify docstring to reflect returned variables (not only total doppler but also `x,` `y` vectors)